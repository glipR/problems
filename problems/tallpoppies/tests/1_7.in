2 64 51
104830097 484655660 884087805 74185788 12946183 253853393 421741424 286656649 669013364 94908054 621692537 800636065 250511900 281645008 78577329 329211451 286745010 306595951 229151357 584012626 86467698 239509690 676338137 193152509 719990522 480498753 545940525 489242275 445558683 519566685 47300237 164865138 474558218 408936641 754724708 774138360 253913742 264814397 743703199 839420651 934953229 218535731 439844334 257037997 613829546 355843080 727556960 675061612 524895897 391033749 288171736 198956337 540839100 98908329 688309635 2732956 929606330 350757662 793469497 995839230 781786225 923548283 228399931 365172439
51471444 1 916263996
55295939 1 869041217
63101808 1 971718930
124154178 0
128293446 1 149834590
137294128 1 60441832
145725742 1 353751115
159514688 0
164392548 0
232999545 0
258139372 0
259632840 1 422714222
283497138 1 630792386
305619040 1 896822736
326504205 1 754771415
338350783 1 993171079
360735812 1 581989471
391169224 0
412951596 1 461931401
414707091 1 930747975
415612120 0
426507808 0
433099693 1 270179712
433320115 0
460939706 1 541808402
462427547 0
480505907 1 492195631
485875337 1 636757427
486639215 0
491421282 0
511323126 1 104424692
552570997 1 547040707
567291798 0
640906386 0
642094677 1 204741654
648922666 0
699209814 0
699605715 1 943462168
704905678 1 71783058
709149116 0
731561268 0
766254668 0
781313094 1 958268170
817996000 1 627329802
819439999 1 411955676
821815891 0
856035863 1 347386627
911143489 0
919126628 0
933461094 0
943234483 0
