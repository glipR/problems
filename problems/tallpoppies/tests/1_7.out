1244147586
1253982605
1258391426
1389547373
1400366549
1627853762
1670644891
1666336724
1667276295
1718324591
1755064655
1756980924
1893032935
2036537480
2041548531
2125976588
2138288754
2151432171
2208847473
2318693394
2491300342
2459945793
2480751734
2470481592
